app/models/channel.rb

```diff
@@ -77,7 +77,9 @@ class Channel < ApplicationRecord
   validate :verified_duplicate_channels_must_be_contested, if: -> { verified? }
 
   after_save :register_channel_for_promo, if: :should_register_channel_for_promo
-  after_save :create_channel_card, :notify_slack, if: -> { :saved_change_to_verified? && verified? }
+  after_save :notify_slack, if: -> { :saved_change_to_verified? && verified? }
+
+  after_commit :create_channel_card, if: -> { :saved_change_to_verified? && verified? }
```

Comment:
I think this possibly could result in a double card create. Might be worth investigating in a redis lock or a sql lock.

Reply:
This method should be able to called multiple times. `UpholdConnectionForChannel` has indexes built in and there is logic to handle duplicates in the `CreateUpholdChannelCardJob`